flow/core/params.py

```diff
@@ -21,6 +21,10 @@ def __init__(self,
                  lateral_resolution=None,
                  no_step_log=True,
                  render=False,
+                 save_render=False,
+                 sight_radius=25,
+                 pxpm=2,
+                 show_radius=False,
```

Comment:
Can we create a separate renderer_params class? Adding it here seems to pollute sumo_params since it isn't a SumoParam

Reply:
The decision was made based on the fact that `save_render` was initially part of SumoParam. Adding a renderer_params will take away `save_render` from SumoParam, which would require changing other parts of Flow. Perhaps we could let this be for the moment and move the task of adding a renderer_params class to an issue?